definitions/npm/validate-currency-code_v0.x.x/flow_v0.104.x-/validate-currency-code_v0.x.x.js

```diff
@@ -0,0 +1,20 @@
+declare module 'validate-currency-code' {
+  declare module.exports: string => boolean;
+}
+
+declare module 'validate-currency-code/codes' {
+  declare module.exports: string[];
+}
+
+declare module 'validate-currency-code/codes.js' {
+  declare module.exports: $Exports<'validate-currency-code/codes'>;
+}
+declare module 'validate-currency-code/index' {
+  declare module.exports: $Exports<'validate-currency-code'>;
+}
+declare module 'validate-currency-code/index.js' {
+  declare module.exports: $Exports<'validate-currency-code'>;
+}
+declare module 'validate-currency-code/test.js' {
+  declare module.exports: $Exports<'validate-currency-code/test'>;
+}
```

Comment:
All those 4 declare are not needed and look like default flow-typed stub generated.